tape/src/main/java/com/squareup/tape/QueueElementObserver.java

```diff
@@ -0,0 +1,26 @@
+package com.squareup.tape;
+
+import java.io.IOException;
+
+public interface QueueElementObserver {
+
+  /**
+   * Observe elements flowing through the Tape queue, to make handling decisions.
```

Comment:
clarify what flowing through means, specifically. ie what specific operations will trigger this.